src/api.js

```diff
@@ -48,7 +48,8 @@ function getPushDetails(eventData) {
   // parsing the ref refs/heads/<branch-name> is the most reliable way
   // to get a branch name
   let branch = ref.split('/').slice(2).join('/');
-  return {
+  let isTagEvent = ref.split('/')[1] === 'tags';
```

Comment:
Let's add a comment here that tags are identified by a ref like `heads/tags/<tagname>`.